HalleDMG/DMGYi135IIIC11c.xml

```diff
@@ -0,0 +1,240 @@
+                        <provenance>The letter is part of the Ethiopianist ‘Nachlass’  of <persName ref="PRS7812Peterman"/>, which
+                            passed after his death to <persName ref="PRS7939Praetori"/> (see    <bibl><ptr target="bm:Praetorius1879Amharisch"/><citedRange unit="page">11</citedRange></bibl>).
+                            </provenance>
+                        <acquisition>It is now kept as part of
+                            the ‘Nachlass Praetorius’ (shelfmark DMG Yi 135) at
+                            <placeName ref="INS0385ULBHalle">Universitäts- und Landesbibliothek Sachsen-Anhalt, Halle (Saale)</placeName> –
+                            <placeName ref="INS0384Bibliothek_der_Deutschen_Morgenlaendischen_Gesellschaft">Bibliothek der Deutschen 
+                                Morgenländischen Gesellschaft</placeName>.</acquisition>
+                    </history>
+                    
+                    <additional>
+                        <adminInfo>
+                            <recordHist>
+                                <source>
+                                  <listBibl type="secondary">
+                                      <bibl><ptr target="bm:Praetorius1879Amharisch"/><citedRange unit="page">11; 506</citedRange></bibl>
+                                      <bibl><ptr target="bm:Guhlmann2023Praetorius"/></bibl>
+                                  </listBibl>
+                                </source>
+                            </recordHist>
+                        </adminInfo>
+                    </additional>
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+            <xi:fallback>
+               <p>Definitions of prefixes used.</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Letter"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="am">Amharic</language>
+                <language ident="ar">Arabic</language>
+                <language ident="de">German</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2024-01-18">Created record after consultation of the manuscript on 21.12.2023</change>
+        </revisionDesc>
+    </teiHeader>
+    <text xml:base="https://betamasaheft.eu/">
+        <body>
+            <div type="edition">
+                <div type="textpart" xml:lang="ar" corresp="#ms_i1.1">
```

Comment:
"am" 1.2?

Reply:
Sorry, I don't understand what you mean - 1.1 is Arabic, 1.2 Amharic

Reply:
Yes but which are you editing below?

Reply:
both